store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9539,4 +9539,9 @@ TODO: delete them permanently from here
 <attr id="3018" name="zimbraPreviousEphemeralBackendURL" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
   <desc>URL of the previous ephemeral storage backend</desc>
 </attr>
+
+<attr id="3019" name="zimbraAttributeMigrationInfo" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
```

Comment:
type `enum` seems more appropriate.
As an aside, some SOAP tests are currently failing because of attempts to modify immutable attributes.  Is there any danger that adding a new one is making that problem worse?  (i.e. are there circumstances where we try to do "get all attrs on an object/update them/put them back" where having some immutable attributes among them causes a problem.